### ISSUE TITLE
Eliminate hardcoding of business_unit (1 & 2) in 4-Projects

### ISSUE THREAD

**Issue by mromascanu123** [terraform-google-modules/terraform-example-foundation#1154]
### TL;DR

- why hardcoded rather than passed via global config via their parameters
- why subnet_ip_range hardcoded?

### Detailed design

```markdown
Provide configurations via top-level global config or at least tfvar-based at the package level
```

**Comment by fmichaelobrien** [comment]
stale bot timer restart - https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/.github/workflows/stale.yml#L21

**Comment by sleighton2022** [comment]
Closing. Our working principal is that this is example code, and not a module. If something needs to be put in place only once, we won't create a variable. If something is used in multiple places, we will use a variable. May be fixed in #1226
